janus_core/cli/geomopt.py

```diff
@@ -171,19 +172,20 @@ def geomopt(
         Device to run model on. Default is "cpu".
     model_path : Optional[str]
         Path to MLIP model. Default is `None`.
-    vectors_only : bool
+    opt_cell_lengths : bool
         Whether to optimize cell vectors, as well as atomic positions, by setting
         `hydrostatic_strain` in the filter function. Default is False.
-    fully_opt : bool
+    opt_cell_fully : bool
         Whether to fully optimize the cell vectors, angles, and atomic positions.
         Default is False.
     filter_func : Optional[str]
         Name of filter function from ase.filters or ase.constraints, to apply
-        constraints to atoms. If using --vectors only or --fully-opt, defaults to
+        constraints to atoms. If using --opt-cell-lengths or --opt-cell-fully, defaults to
         `FrechetCellFilter` if available, otherwise `ExpCellFilter`.
```

Comment:
```suggestion
        constraints to atoms. If using --opt-cell-lengths or --opt-cell-fully, defaults
        to `FrechetCellFilter` if available, otherwise `ExpCellFilter`.
```